<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PenguinSettings</class>
 <widget class="QWidget" name="PenguinSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Penguin Settings</string>
  </property>
  <widget class="QCheckBox" name="askToClose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ask to close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="logLocation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Window notification</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Status bar</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Log location:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Penguin v1</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/swiftshine/penguin&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Penguin on Github&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="maintainPosition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>If checked, the window will keep the last position it was at.</string>
   </property>
   <property name="text">
    <string>Maintain window position</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
